InstagramApp/InstagramApp/MainScreenViewController.swift

```diff
@@ -0,0 +1,83 @@
+//
+//  ViewController.swift
+//  InstagramApp
+//
+//  Created by Daniil on 17.10.2022.
+//
+
+import UIKit
+
+// Основная лента с фото
+final class MainScreenViewController: UIViewController {
+
+    // MARK: - IBOutlets
+    @IBOutlet weak var tableView: UITableView!
```

Comment:
```suggestion
    @IBOutlet weak var tableView: UITableView!
```
private у всех атлетов